src/pages/management/configuration-api/v3.6/index.mdx

```diff
@@ -3059,9 +3059,10 @@ curl -X POST \
 
 ### Enable License Webhooks
```

Comment:
I was wondering - in all the other versions, there is also a change in the Register Webhook method - shouldn't it be made in this version as well?

Reply:
had the changes locally, forgot to save the file 🤦